### DISCUSSION TITLE
External Micro USB Port for Updates

### DISCUSSION THREAD

**Discussion by Addonnis** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#420] [Show and tell]
Added an external Micro USB port for the ESP, less case opening the better IMO!

Search eBay for **Micro USB M/F Mount Extension Dash Flush Dashboard Cable for Car Boat Motorcycle**

![image](https://user-images.githubusercontent.com/97511696/208840173-fea3ec13-ede8-4661-8c76-31b38a38f824.png)

![image](https://user-images.githubusercontent.com/97511696/208840231-d4c83541-83a7-4016-8f78-a8e11fc2fcd8.png)

**Comment by torei** [comment]
cute :D
did you ever think about OTA updates?

**Reply by Addonnis** [reply]
I have! But I haven't been able to get OTA updates working reliably with each release (probably a "me" problem) but I like the OEM+ aesthetic and it's a cheap mod, so why not.
